MapleServer2/PacketHandlers/Game/InsigniaHandler.cs

```diff
@@ -1,7 +1,10 @@
-﻿using MaplePacketLib2.Tools;
```

Comment:
If we're calling this file InsigniaHandler and we know the opcode is only used for insignias, let's change the recv and send ops to `INSIGNIA`